cluster/terraform_kubernetes/config/test.tfvars.json

```diff
@@ -2,7 +2,9 @@
   "cip_tenant": true,
   "namespaces": [
     "bat-qa",
-    "bat-staging"
+    "bat-staging",
+    "tra-dev",
```

Comment:
As discussed, please change tra-dev to tra-development

Reply:
Update completed.